deploy/olm-catalog/qdr-operator/0.1.0/catalog-source.yaml

```diff
@@ -48,50 +49,88 @@ items:
                   version: v1
                 - kind: secrets
                   version: v1
+                - kind: customresourcedefinitions
+                  version: v1beta1
                 specDescriptors:
                 - description: The role and placement plan for the interconnect deployment
-                  displayName: DeploymentPlan
-                  path: deploymentPlan
+                  displayName: Deployment Plan Image
+                  path: deploymentPlan.image
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
+                - description: The role and placement plan for the interconnect deployment
+                  displayName: Deployment Placement
+                  path: deploymentPlan.placement
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
+                - description: The role and placement plan for the interconnect deployment
+                  displayName: Deployment Role
+                  path: deploymentPlan.role
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
+                - description: Listeners for incoming connections to the router
+                  displayName: Listener Port
+                  path: listeners.port
```

Comment:
I don't think displaying the first listener port is particularly useful. There doesn't seem to be a way to handle lists effectively yet. From what I can see this mainly affects the overview page in the openshift console. I'd be inclined to keep things simple and just display the role and placement (since those are really the defining characteristics of the deployment) for the time being.

Reply:
![podstaus](https://user-images.githubusercontent.com/42355256/61244680-bfc62b80-a718-11e9-8018-79daf4b74a81.png)

we will display role and placement. I took out others.